desktop/libs/notebook/src/notebook/templates/editor_components.mako

```diff
@@ -2088,6 +2088,12 @@ ${ sqlSyntaxDropdown.sqlSyntaxDropdown() }
         snippetImage: '${ static("beeswax/art/icon_beeswax_48.png") }',
         sqlDialect: true
       },
+      hplsql: {
+        placeHolder: '${ _("Example: SELECT * FROM tablename, or press CTRL + space") }',
```

Comment:
How about an HPL/SQL example here? For instance: `placeHolder: 'Example: PRINT \'Hello world\''`